src/Webapi/Webapi__Dom/Webapi__Dom__Window.re

```diff
@@ -77,7 +77,7 @@ module Impl = (T: {type t;}) => {
   [@bs.send.pipe : t_window] external matchMedia : string => mediaQueryList = ""; /* experimental, CSSOM View module */
   [@bs.send.pipe : t_window] external moveBy : (int, int) => unit = ""; /* experimental, CSSOM View module */
   [@bs.send.pipe : t_window] external moveTo : (int, int) => unit = ""; /* experimental, CSSOM View module */
-  [@bs.send.pipe : t_window] [@bs.return nullable] external open_ : (~url: string, ~name: string, ~features: string) => option(Dom.window) = "open"; /* yes, features is a stringly typed list of key value pairs, sigh */
+  [@bs.send.pipe : t_window] [@bs.return nullable] external open_ : (~url: string, ~name: string, ~features: option(string)) => option(Dom.window) = "open"; /* yes, features is a stringly typed list of key value pairs, sigh */
```

Comment:
This isn't how you make arguments optional. I think the proper syntax is `~features: string=?`